network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -533,7 +533,7 @@ def exec(self):
         return res
 
     def compute_query_output(self, query_answer):
-        return self.get_query_output(query_answer, query_answer.bool_result)
+        return self.get_query_output(query_answer, only_explanation=query_answer.bool_result)
```

Comment:
this is exactly the same as before, I added the flag name only to prevent confusion